src/nestjs-back/src/main.ts

```diff
@@ -42,7 +42,7 @@ async function bootstrap() {
     }),
   );
 
-  seeder();
+  // seeder();
```

Comment:
Remember to uncomment this to get achievements generated before going to defense (and think about seeding fake users, gales...)